exercises/roman-numerals/src/example.c

```diff
@@ -0,0 +1,43 @@
+#include "roman_numerals.h"
+#include <stdlib.h>
+#include <string.h>
+
+#define NUM_OF_ELEMENTS(a)    (sizeof(a) / sizeof(a[0]))
+#define MAX_NUMERAL_LENGTH    (6)
+
+typedef struct
+{
+    char *numeral;
+    unsigned int value;
+} Numeral_Values_t;
```

Comment:
I don't think we've defined our own types in our style conventions. My usual strategy is to use Caps for user types to distinguish it from library types, so I have a slight preference for `Numeral_Values_t` or even `NumeralValues_t` in this case.

It obviously isn't really important to me either way.